Lib/idlelib/configdialog.py

```diff
@@ -650,83 +650,85 @@ def create_page_keys(self):
         frames[1].pack(side=TOP, fill=X, expand=True, pady=2)
         return frame
 
+
     def create_page_general(self):
         """Return frame of widgets for General tab.
 
-        Tk Variables:
-            win_width: Initial window width in characters.
-            win_height: Initial window height in characters.
-            startup_edit: Selector for opening in editor or shell mode.
-            autosave: Selector for save prompt popup when using Run.
-
-        Methods:
-            load_general_config:
-            help_source_selected: Bound to list_help button release.
-            set_helplist_button_states: Toggle based on list.
-            helplist_item_edit: Command for button_helplist_edit.
-            helplist_item_add: Command for button_helplist_add.
-            helplist_item_remove: Command for button_helplist_remove.
-            update_user_help_changed_items: Fill in changes.
+        Enable users to provisionally change general options.
+        Load_general_cfg loads current values.  Radiobuttons
+        startup_shell_on and startup_editor_on set var startup_edit.
```

Comment:
I used 'sets var x_y' in the FontTab docstring.  I tried Var instead of var.  We could use 'tk variable' to be really explicit, but I was hoping 'var' or 'Var' as an abbreviation (StringVar, IntVar, etc) would be clear enough.  I am going to push as is, but we can revisit this in 'extract class' issues where function docstring become class docstring.  bpo-31004, bpo-31050